xDripG5/Transmitter.swift

```diff
@@ -8,14 +8,79 @@
 
 import Foundation
 import CoreBluetooth
+import HealthKit
 import os.log
 
 
+public enum Command: RawRepresentable {
+    case startSensor(at: Date)
+    case stopSensor(at: Date)
+    case calibrateSensor(to: HKQuantity, at: Date)
+
+    public typealias RawValue = [String: Any]
+
+    public init?(rawValue: RawValue) {
+        guard let command = rawValue["command"] as? Int else {
+            return nil
+        }
+
+        switch command {
+        case 0:
```

Comment:
`0`, `1`, and `2` shouldn't be used as free-floating values. Define a private, Int-backed enum. If `Command` continues to expand, it should probably be expanded into a protocol instead of a single enum.

Reply:
Thanks. I've added an Action enum. Please advise if the implementation isn't ideal.